src/Compilers/CSharp/Test/Emit2/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -178,6 +184,22 @@ public void PubKeyFromKeyFileAttribute_AssemblyKeyFileResolver_RelativeToCurrent
             Assert.True(ByteSequenceComparer.Equals(s_publicKey, comp.Assembly.Identity.PublicKey));
         }
 
+        [Fact]
+        public void PubKeyFromKeyFileAttribute_SigningTempPathNotAvailable()
+        {
+            var x = s_keyPairFile;
```

Comment:
Consider using a more descriptive variable name.

Reply:
removed the variable as I didn't see aneed for it anymore.